pkg/controller/networkpolicyrecommendation/controller.go

```diff
@@ -194,6 +199,10 @@ func (c *NPRecommendationController) Run(stopCh <-chan struct{}) {
 		return
 	}
 
+	// The key can be anything as we only have single item.
+	c.gcQueue.Add("key")
```

Comment:
Maybe create a const for the key in this package and use that?

Reply:
Updated the key to a specific gcKey to allow separation, thanks!